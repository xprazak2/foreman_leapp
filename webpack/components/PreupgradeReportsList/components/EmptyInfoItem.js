import React from 'react';
import PropTypes from 'prop-types';

import InfoItem from './InfoItem';
import { itemIteratorId } from './helpers';

const EmptyInfoItem = ({ entry, attr }) => (
  <InfoItem itemId={itemIteratorId(entry, attr)} />
);

EmptyInfoItem.propTypes = {
  entry: PropTypes.object.isRequired,
  attr: PropTypes.string.isRequired,
};

export default EmptyInfoItem;
