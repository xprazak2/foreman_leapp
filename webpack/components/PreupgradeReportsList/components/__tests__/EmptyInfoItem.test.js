import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import EmptyInfoItem from '../EmptyInfoItem';

const fixtures = {
  'should render': {
    entry: { title: 'no title', severity: 'low', id: 5, preupgradeReportId: 7 },
    attr: 'title',
  },
};

describe('EmptyInfoItem', () =>
  testComponentSnapshotsWithFixtures(EmptyInfoItem, fixtures));
