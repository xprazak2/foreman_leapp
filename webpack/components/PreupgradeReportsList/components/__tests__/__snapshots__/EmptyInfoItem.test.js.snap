// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyInfoItem should render 1`] = `
<InfoItem
  itemId="5-7-title"
  tooltipText=""
/>
`;
